sphinx/source/docs/releases/2.4.0.rst

```diff
@@ -32,6 +32,7 @@ Other work in this cycle includes:
     - Clipping area problem fixed :bokeh-issue:`11551`
 
 * Misc fixes and improvements
+    - Labels can now be multi-line :bokeh-issue:`7317`
```

Comment:
Perhaps we could mention `Title` and `LabelSet` as well (especially the former).

Reply:
👍  I was not aware I will add this

Reply:
https://github.com/bokeh/bokeh/pull/11599/commits/89290caee81e32c6f6a3bfc72171c479d974cc28